packages/store/src/commonSelectors.ts

```diff
@@ -215,12 +215,9 @@ export const getAvailableIssueOptionsByType = createSelector(
     }, {})
 );
 
-export const getGroupNames = createSelector([getGroupsById, getUserRoles, (_, options = {}) => options], (groupsById, { uiPermissions }, { staticOnly }) => {
+export const getGroupNames = createSelector([getGroupsById, getUserRoles], (groupsById, { uiPermissions }) => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { [UNGROUPED_GROUP.id]: ungrouped, ...groups } = groupsById;
-  if (staticOnly) {
-    return Object.keys(uiPermissions.groups).sort();
-  }
```

Comment:
that seems to be a change in functionality and not a purely test related change

Reply:
based on the commit message it is a change to the test - which it is clearly not.